master/buildbot/newsfragments/issue4187-gitlab-merge-request-get-tags-from-target-too.bugfix

```diff
@@ -0,0 +1 @@
+-:bb:step:`GitHub` now also fetches keys from target git repo when building merge requests. (:issue:`4187`)
```

Comment:
File has `gitlab` in it but here it's `GitHub`.

Reply:
Whoops.